code/modules/reagents/oldchem/reagents/_reagent_base.dm

```diff
@@ -16,7 +16,7 @@
 	//Processing flags, defines the type of mobs the reagent will affect
 	//By default, all reagents will ONLY affect organics, not synthetics. Re-define in the reagent's definition if the reagent is meant to affect synths
 	var/process_flags = ORGANIC
-
+	var/adminOnly = 0
```

Comment:
Yeah, you're right. I default to camelcase, but where underscores are already established I should be using them.

I'll fix it now